src/dotnet/commands/dotnet-pack/LocalizableStrings.cs

```diff
@@ -2,29 +2,29 @@
 {
     internal class LocalizableStrings
     {
-        public const string AppFullName = "pack";
+        public const string AppFullName = ".NET Core NuGet Package Packer";
 
         public const string AppDescription = "pack for msbuild";
 
         public const string CmdOutputDir  = "OUTPUT_DIR";
 
-        public const string CmdOutputDirDescription = "Directory in which to place outputs";
+        public const string CmdOutputDirDescription = "Directory in which to place built packages.";
 
-        public const string CmdNoBuildOptionDescription = "Do not build project before packing";
+        public const string CmdNoBuildOptionDescription = "Skip building the project prior to packing. By default, the project will be built.";
 
-        public const string CmdIncludeSymbolsDescription = "Include PDBs along with the DLLs in the output folder";
+        public const string CmdIncludeSymbolsDescription = "Include packages with symbols in addition to assembly (DLL) packages in output directory.";
 
         public const string CmdIncludeSourceDescription  = "Include PDBs and source files. Source files go into the src folder in the resulting nuget package";
 
         public const string CmdConfig = "CONFIGURATION";
 
-        public const string CmdConfigDescription = "Configuration under which to build";
+        public const string CmdConfigDescription = "Configuration to use for building the project.  Default is  \"Debug\".";
```

Comment:
This may be misleading. I believe we set the default configuration in the SDK and it can be overridden by targets/props. Consider using `Default for most projects is \"Debug\".`